compiler/qsc_frontend/src/typeck/convert.rs

```diff
@@ -256,11 +256,8 @@ pub(super) fn scheme_for_ast_callable(
     names: &Names,
     callable: &CallableDecl,
 ) -> (Scheme, Vec<TyConversionError>) {
-    let (mut type_parameters, errors) = type_parameters_for_ast_callable(names, &callable.generics);
-    let mut errors = errors
-        .into_iter()
-        .map(TyConversionError::from)
-        .collect::<Vec<_>>();
+    let (mut type_parameters, mut errors) =
+        type_parameters_for_ast_callable(names, &callable.generics);
```

Comment:
Clippy recommended just removing line 262, but then it's just `errors().into_iter().collect::<Vec<_>>();`, and according to the vscode type hints, `errors` already has the target type - it just needed to be mutable:

![Screenshot from 2025-03-28 16-19-04](https://github.com/user-attachments/assets/6a574ce7-9120-4fb8-b684-2c6b854cde60)

Regarding the line-break... prettier (which is included by the project) appears to use 80chars as a default... it actually additionally changed a bunch of docstrings as well, but I didn't include those to focus on the actual change here. But let me know if it shouldn't be wrapped here.